src/Sylius/Behat/Context/Api/Shop/LoginContext.php

```diff
@@ -61,7 +61,7 @@ public function iWantToLogIn(): void
      */
     public function iWantToResetPassword(): void
     {
-        $this->request = Request::create('shop', 'request-reset-password', 'Bearer');
+        $this->request = Request::create('shop', 'reset-password-requests', 'Bearer');
```

Comment:
Well I would call here also @GSadee as, first version was `reset-password-request`, then it was suggested to change to `reqest-reset-password` and PATCH `reset-password/{token}` now it is changed to `reset-password-requests` (plural form), and from what I see in our task there was suggested to use singular `reset-password-request`

Reply:
To be honest, I got a little lost in what these endpoints should look like, the current form is consistent with endpoints like `apply-coupon` or `change-quantity`, but these ones should be based on the API resource, so this is a good change.